piptools/repositories/pypi.py

```diff
@@ -378,6 +378,8 @@ def _get_hashes_from_files(self, ireq: InstallRequirement) -> set[str]:
 
     def _get_file_hash(self, link: Link) -> str:
         log.debug(f"Hashing {link.show_url}")
+        if link.hash_name == FAVORITE_HASH:
+            return ":".join([FAVORITE_HASH, link.hash])
```

Comment:
```suggestion
        if link.hash_name == FAVORITE_HASH:
            log.debug(f"Getting hash from link {link.show_url}")
            return ":".join([FAVORITE_HASH, link.hash])
        log.debug(f"Hashing {link.show_url}")
```

Maybe different log message?